src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -59,6 +60,22 @@ internal static class ProtocolConversions
             { WellKnownTags.NuGet, LSP.CompletionItemKind.Text }
         };
 
+        public static Uri GetUriFromFilePath(string filePath)
+        {
+            if (filePath is null)
+            {
+                throw new ArgumentNullException(nameof(filePath));
+            }
+
+            // Remove preceding slash if we're on Window as it's an invalid URI.
+            if (filePath.StartsWith("/") && RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
To clarify, so what is the paths we're getting from Razor? It's like /C:/foo/bar/baz.txt? Or something else?

Reply:
You got it exactly!